action.yml

```diff
@@ -28,18 +37,27 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - id: go-critic
+      run: echo "version=v0.8.1" >> $GITHUB_OUTPUT
+      shell: bash
     - id: go-junit-report
       run: echo "version=v2.0.0" >> $GITHUB_OUTPUT
       shell: bash
     - id: gomock
       run: echo "version=v1.6.0" >> $GITHUB_OUTPUT
       shell: bash
+    - id: gosec
+      run: echo "version=v2.16.0" >> $GITHUB_OUTPUT
+      shell: bash
     - id: packer
       run: echo "version=1.8.2" >> $GITHUB_OUTPUT
       shell: bash
     - id: shfmt
       run: echo "version=v3.5.1" >> $GITHUB_OUTPUT
       shell: bash
+    - id: staticcheck
+      run: echo "version=2023.1.3" >> $GITHUB_OUTPUT
```

Comment:
Whichever version of staticcheck you end up on, there is some commit history to be cleaned up here.

Reply:
@jsf9k do you mean combine them into one to remove duplicate commits?

Reply:
- Commit 265128b changes the version from `v0.4.3` to `2023.1.3`
- Commit 9fe08e9 changes the version from `2023.1.3` to `v0.4.3`
- Commit 73eb498 changes the version from `v0.4.3` to `2023.1.3`

So you could rebase and drop commits 9fe08e9 and 73eb498 with no change to the actual code.

Reply:
Although I see that @mcdonnnj prefers the semver version (as do I), so if you follow that advice you could rebase and drop all three commits.

Reply:
@jsf9k  Please see https://github.com/cisagov/setup-env-github-action/pull/65/commits/c625d814f0afdbed58563decc5ad19bc2fdabba0